py/webskylpt/toy_model_code/light_cone.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np 
+import cosmology as cosmo
+import healpy as hp
+import matplotlib.pyplot as plt
+import scipy.constants as cons
+# from joblib import Parallel, delayed
+from mpi4py import MPI
+from time import time
+
+# MPI communicator initialization
```

Comment:
@1cosmologist can you put in some mechanism to allow the light cone projection to run in serial mode without MPI?

Reply:
Alright, I will put in some switches to change between MPI and serial compute.

Reply:
Thanks, hopefully that's helpful for testing.